apps/condo/domains/banking/access/BankAccount.js

```diff
@@ -30,11 +54,33 @@ async function canReadBankAccounts ({ authentication: { item: user } }) {
  * BankAccount can be managed only by:
  * 1. Admin or support
  */
-async function canManageBankAccounts ({ authentication: { item: user } }) {
+async function canManageBankAccounts ({ authentication: { item: user }, context, operation, originalInput, itemId }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
     if (user.isAdmin || user.isSupport) return true
 
+    if (user.type === SERVICE) {
+        if (await checkBankIntegrationsAccessRights(context, user.id, [BANK_INTEGRATION_IDS.SBBOL])) {
+
+            if (operation === 'update') {
+                const bankAccount = await getById('BankAccount', itemId)
+                const integrationContext = get(bankAccount, 'integrationContext')
+
+                if (integrationContext) {
+                    const accountContext = await getById('BankIntegrationAccountContext', integrationContext)
+
+                    if (accountContext.integration !== BANK_INTEGRATION_IDS.SBBOL) return false
```

Comment:
You should return false by default!
```suggestion
                    if (accountContext.integration === BANK_INTEGRATION_IDS.SBBOL) return true
                    return false
```